model/project_ref.go

```diff
@@ -2032,6 +2032,19 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 		if catcher.HasErrors() {
 			return false, errors.Wrapf(catcher.Resolve(), "validating external links")
 		}
+		var pRef *ProjectRef
+		pRef, err = FindBranchProjectRef(projectId)
+		if err != nil {
+			return false, errors.Wrapf(err, "error getting project")
+		}
+		if pRef == nil {
+			return false, errors.Errorf("project '%s' was not found", projectId)
+		}
+		// If the performance plugin is not currently enabled, and we are trying to
+		// change it to enabled but the id and identifier ar edifferent, no-op.
```

Comment:
Also we are erroring here, not no-opping if the id and identifier are different so we should change the comment above